pkg/kapp/cmd/app/deploy.go

```diff
@@ -92,6 +92,7 @@ func NewDeployCmd(o *DeployOptions, flagsFactory cmdcore.FlagsFactory) *cobra.Co
 }
 
 func (o *DeployOptions) Run() error {
+	var newApp bool
```

Comment:
We can declare this inline

Reply:
Can not declare  inline as variable is being declared inside if block.